src/app/code/community/AvS/FastSimpleImport/Model/Import/Entity/Category.php

```diff
@@ -548,16 +575,24 @@ protected function _saveCategories()
                         'level'       => $parentCategory['level'] + 1,
                         'created_at'  => empty($rowData['created_at']) ? now()
                                 : gmstrftime($strftimeFormat, strtotime($rowData['created_at'])),
-                        'updated_at'  => now(),
-                        'position'    => $rowData['position']
+                        'updated_at'  => now()
                     );
 
                     if (isset($this->_categoriesWithRoots[$rowData[self::COL_ROOT]][$rowData[self::COL_CATEGORY]]))
                     { //edit
-
                         $entityId = $this->_categoriesWithRoots[$rowData[self::COL_ROOT]][$rowData[self::COL_CATEGORY]]['entity_id'];
+                        $path = $parentCategory['path'] .'/'.$entityId;
+                        $position = $this->_categoriesWithRoots[$rowData[self::COL_ROOT]][self::COL_CATEGORY]['position'];
```

Comment:
I think the second level key in the array was meant to be `$rowData[[self::COL_CATEGORY]]`